recipes/date/all/conanfile.py

```diff
@@ -23,7 +35,51 @@ def source(self):
         for patch in self.conan_data["patches"][self.version]:
```

Comment:
Prefer patching source in build.

Reply:
Okay, moved just to the beginning of the build - is there reasoning behind this?
It seems that patching in the source would make more sense ( as it would only be done together with source ), having it in build, causes issues when invoking build multiple times after a single conan source.

Reply:
I looked at the recipes out there - it seems that more of them patch in build actually.. okay.